caravel/views.py

```diff
@@ -1793,6 +1793,12 @@ def dashboard(self, dashboard_id):
         def dashboard(**kwargs):  # noqa
             pass
         dashboard(dashboard_id=dash.id)
+        if request.args.get("standalone") == "true":
+            return self.render_template(
+                "caravel/dashboard_standalone.html",
```

Comment:
do we need to render a new template for the standalone view? or could we add some logic to `basic.html` where if we are on the dashboard page and the standalone arg is true, don't show the header. we are doing something similar for the standalone viz's afaik. 

https://github.com/ascott/caravel/blob/master/caravel/templates/caravel/basic.html#L24-L25